src/docfx/build/toc/BuildTableOfContents.cs

```diff
@@ -54,7 +63,7 @@ private static void BuildTocMap(Context context, Document fileToBuild, TableOfCo
                 Debug.Assert(tocMapBuilder != null);
                 Debug.Assert(fileToBuild != null);
 
-                var (errors, tocModel, _, referencedDocuments, referencedTocs) = Load(context, fileToBuild);
+                var (_, _, _, referencedDocuments, referencedTocs, _) = Load(context, fileToBuild, null);
```

Comment:
I think we still need to report any errors here, errors will be de-duped if they are the same.